components/forms/Login/index.jsx

```diff
@@ -36,26 +33,16 @@ const Login = () => {
         >
           Password
         </label>
-        <input
-          className="w-full h-14 bg-white/20 text-white text-base-large rounded-xl px-3 py-1.5 pr-[23%] focus:outline-none focus:shadow-none"
-          id="password"
-          data-test-id={loginFormTestIds.passwordField}
-          type="password"
-          required
-        />
+        <Input id="password" type="password" dataTestId={loginFormTestIds.passwordField} required={true} className="pr-[23%]" />
         <Link href="/reset-password">
-          <a className="absolute text-white/50 text-base-small right-3 bottom-[18px]">Forgot?</a>
+          <a className="absolute text-white/50 text-base-small right-3 bottom-4">Forgot?</a>
         </Link>
       </div>
-      <button
-        className="w-full h-14 bg-white text-base-large font-extrabold rounded-xl"
-        data-test-id={loginFormTestIds.loginButton}
-        type="submit"
-      >
+      <Button type="submit" size="medium" dark={true} dataTestId={loginFormTestIds.loginButton}>
```

Comment:
Same here 🙌🏼 

```suggestion
      <Button type="submit" size="medium" dark dataTestId={loginFormTestIds.loginButton}>
```

Btw does the dark style is intentionally added for the disabled button? cuz I cannot find them in Figma
And if it's that so maybe using the `disabled` property might be more make sense IMO 😄

Reply:
@carryall, this particular prop should be `false` actually, I was testing out the `dark={true}` prop and didn't undo 😶 
Regarding what's the use case, it adds a dark background class. Here it is on figma(yes button)

![image](https://user-images.githubusercontent.com/39977805/160981576-bf044024-e1f8-477d-8609-bd4d207771a2.png)